modules/mocked_authentication/app/services/mocked_authentication/mockdata_fetcher.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'mockdata/reader'
+
+module MockedAuthentication
+  class MockdataFetcher
```

Comment:
Do we need this class? Seems like it's more or less an alias of `MockData::Reader`?